src/components/Icon/IconChevron.tsx

```diff
@@ -7,15 +7,20 @@ import cn from 'classnames';
 
 export const IconChevron = memo<
   JSX.IntrinsicElements['svg'] & {
-    displayDirection: 'up' | 'down' | 'left' | 'right';
+    /**
+     * The direction the arrow should point.
+     * `start` and `end` are relative to the current locale.
+     * for example, in LTR, `start` is left and `end` is right.
+     */
+    displayDirection: 'start' | 'end' | 'right' | 'left' | 'up' | 'down';
   }
 >(function IconChevron({className, displayDirection}) {
   const classes = cn(
     {
       'rotate-0': displayDirection === 'down',
-      'rotate-90': displayDirection === 'left',
+      'rotate-90': displayDirection === 'left' || displayDirection === 'start',
```

Comment:
Why didn't we create a separate `start` key here? re:
`'rotate-90 rtl:-rotate-90': displayDirection === 'start'`

Reply:
Nice catch
 resolved